trace/filtered_export.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trace
+
+import "strings"
+
+// SpanFilter allows matching SpanData by well-defined criteria.
+type SpanFilter struct {
+	// we deliberately keep all SpanFilter implementations known in this
+	// package so it's easier for us to define them in configuration in future.
+	fn func(*SpanData) (shouldExport bool)
```

Comment:
You probably shouldn't call this `shouldExport`, since you use the opposite meaning in `ExcludingSpans`.

I think you shouldn't name the return value at all. And maybe even call this `SpanMatcher`, since it's really agnostic about how it's used.